controllers/database_controller.go

```diff
@@ -418,7 +438,11 @@ func (r *DatabaseReconciler) reconcilePSMDB(ctx context.Context, req ctrl.Reques
 		}
 		if database.Spec.Backup != nil {
 			if database.Spec.Backup.Image == "" {
-				database.Spec.Backup.Image = fmt.Sprintf(psmdbBackupImageTmpl, psmdb.Spec.CRVersion)
+				image, err := version.PSMDBBackupImage()
+				if err != nil {
+					return err
+				}
+				database.Spec.Backup.Image = image
```

Comment:
Why are we only doing this for PSMDB and kept the old Sprintf fox PXC?

Reply:
Because image comes from version service only for psmdb operator